experiment.py

```diff
@@ -151,15 +154,21 @@ def run(config_file:str, logging_level:str=DEFAULT_LOGLEVEL):
     bas_min = float(config.getValue("Experiments", "bas_min"))
     bas_max = float(config.getValue("Experiments", "bas_max"))
     bas_step = float(config.getValue("Experiments", "bas_step"))
+    end_of_phrase_silence_time_min = float(config.getValue("Experiments", "end_of_phrase_silence_time_min"))
+    end_of_phrase_silence_time_max = float(config.getValue("Experiments", "end_of_phrase_silence_time_max"))
+    end_of_phrase_silence_time_step = float(config.getValue("Experiments", "end_of_phrase_silence_time_step"))
 
     custom_model = str(config.getValue("SpeechToText", "language_model_id"))
     
     bias_range = drange(bias_min, bias_max+bias_step, bias_step)
     weight_range = drange(cust_weight_min, cust_weight_max+cust_weight_step, cust_weight_step) if custom_model!="None" else drange(0.0, 0.1, 0.1)    
     sds_range = drange(sds_min, sds_max+sds_step, sds_step) 
     bas_range = drange(bas_min, bas_max+bas_step, bas_step)
+    end_of_phrase_silence_time_range = drange(end_of_phrase_silence_time_min,end_of_phrase_silence_time_max+end_of_phrase_silence_time_step,end_of_phrase_silence_time_step)
     
-    experiments.run_all_experiments(bias_range, weight_range, sds_range, bas_range, max_threads, logging_level)
+    print(sds_range)
```

Comment:
is this print(sds_range) supposed to be in there?

Reply:
good catch, no. Removed it